lib/internal/process/policy.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const {
+  ERR_MANIFEST_TDZ,
+} = require('internal/errors').codes;
+const fs = require('fs');
+const { pathToFileURL } = require('internal/url');
+const { URL } = require('url');
+const { Manifest } = require('internal/policy/manifest');
+const { getOptionValue } = require('internal/options');
+let manifest;
+module.exports = Object.freeze({
+  __proto__: null,
+  setup() {
+    const manifestSpecifiers = getOptionValue('--experimental-policy');
+    if (manifestSpecifiers.length === 0) {
+      manifest = null;
+      return;
+    }
+    const cwdURL = pathToFileURL(process.cwd());
```

Comment:
Can you turn the setup in this file as side effect free as possible and make the side effect more explicit in `bootstrap/node.js`? Hiding the side effects and file path resolution in a separate file makes it difficult to tell if this side effect is reproducible when loading from a snapshot and if it needs special care in workers or child processes (that all run `bootstrap/node.js` but may have different cwd). From reading the code, I don't think the current implementation has taken any of those complexities into account?

Reply:
I would not expect snapshots to contain policies, but I can move things if I get help. I don't fully understand the request.

Reply:
It would be helpful to make the calls to this look something like

```js
const experimentalPolicy = getOptionValue('--experimental-policy');
if (experimentalPolicy) {
  const policy = NativeModule.require('internal/process/policy');
  const manifest = policy.load(process.cwd(), experimentalPolicy);
  policy.setManifest(manifest);
}
```

So that we'll know this branch needs special care in the case of

- Snapshot loading
- Clusters, workers, child processes

in `bootstrap/node.js` and handle them correctly (e.g. postpone the resolution to a point where environment-dependent settings are safe to be loaded). Otherwise this would be a hidden bomb when we start building environment-independent snapshots.

Reply:
moved IO to node.js so it is readily visible